src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -107,48 +100,60 @@ export default class PhotoField extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    const newView = this.state.src && (prevState.src !== this.state.src || (prevState.done && !this.state.done));
+    const newView = this.state.src && (prevState.src !== this.state.src || ((this.state.progress === 100) && (this.state.progress < 100)));
```

Comment:
`((this.state.progress === 100) && (this.state.progress < 100))` 
I think this will always evaluate to false.

Reply:
Great catch, thanks @rianfowler .